Misc/NEWS.d/next/Tests/2018-09-18-12-03-41.bpo-34720.d5uOoP.rst

```diff
@@ -0,0 +1 @@
+Fix `test_importlib.test_bad_traverse` for AIX
```

Comment:
Test-only changes usually don't require a NEWS entry. In this case, it should be removed.

(Bedevere will complain, but you may ignore that, and I or another core dev will simply add the "skip news" tag.)